docs/modules/ROOT/pages/explanations/policies.adoc

```diff
@@ -0,0 +1,314 @@
+
+=== Request flow and policy execution
+
+The diagram shows how the Kyverno policies interact with a request to create a `Namespace`, `Project` or `ProjectRequest`.
+
+[IMPORTANT]
+====
+We only show interactions which are relevant to `CREATE` requests.
+Some policies, such as `validate-namespace-metadata` are also executed on `UPDATE` requests.
+
+See the detailed policy explanations or the policy reference documentation for details.
+====
+
+[ditaa, namespace-policies, svg]
+....
+                      /---------------\
+                      | cFD0          |
+                      | CREATE        |
+                      | Namespace,    |
+                      | ProjectRequest|
+                      |               |
+                      \-------+-------/
+                              |
+                              |
+                   PRIV       v
+                   +---------------------+
+                   |  +---------------+  |
+                   |  |cBLUVALIDATE   |  |
+                   |  +---------------+  |    yes
+                   | Principal is allowed+------------------------------------------+
+                   |  to bypass policies |                                          |
+                   | {d}                 |                                          |
+                   +----------+----------+                                          |
+                              |                                                     |
+                              |no                                                   |
+                              |                                                     |
+                   RSVD       v                                                     |
+                   +---------------------+                                          |
+                   |  +---------------+  |                                          |
+                   |  |cBLUVALIDATE   |  |                                          |
+                   |  +---------------+  |                                          |
+                   | Check that requested|                                          |
+                   | namespace/project   |                                          |
+                   | name doesn't match a|                                          |
+                   | reserved prefix     |                                          |
+                   | {d}                 |                                          |
+                   +----------+----------+                                          |
+                              |                                                     |
+            +-----------------+-----------------------+                             |
+            |                                         |                             |
```

Comment:
Very nitpicky: There should be a `no` option. (But that might actually be more confusing than helpful because too many arrows :sweat_smile: )

Reply:
Hm, that was there at some point but apparently got lost in one of the diagram refactorings

Reply:
Added an arrow going to `DENY` (and made the `disallow-reserved-namespaces` description less confusing)